### PR TITLE
Disable network manager to prevent it from adding dns entries in /etc…

### PR DESCRIPTION
**Pull request by Xenwar**
Disable network manager to prevent it from adding dns entries in /etc/resolv.conf

### CODE REVIEW

**Review by namnx228 — APPROVED** [review]
LGTM

**Review by smoshiur1237 — COMMENTED** [review]
/lgtm

**Comment by smoshiur1237** [comment]
I think you need to do a signed commit to get this merged.

File: ci/images/userdata.tpl [hunk 1]
```diff
@@ -9,3 +9,5 @@ users:
 
 runcmd:
   - sed -i "/^127.0.0.1/ s/$/ ${HOSTNAME}/" /etc/hosts
+  - systemctl stop NetworkManager.service
+  - systemctl mask NetworkManager.service
```
Comment (furkatgofurov7) [inline ci/images/userdata.tpl:13]: Wondering how was this verified?

**Comment by Xenwar** [comment]
This PR is not longer needed. 
The issue is handled by https://github.com/Nordix/airship-dev-tools/pull/302
Closing it.

File: ci/images/userdata.tpl (hunk 1 above)
Comment (Xenwar) [reply ci/images/userdata.tpl:13]: PR is closed. Adding a comment for future reference. 
Disabling the network manager did not help as it made the VM not accessible via ssh.
The alternative was to make the ```/etc/resolv.conf``` file immutable. see here https://github.com/Nordix/airship-dev-tools/pull/302

**Pull request closed without merging**